bsip-0061.md

```diff
@@ -75,3 +75,8 @@ This document is created for the betterment of humanity and is hereby placed int
 # Discussion
 A full implementation is available at:
 https://github.com/bitshares/bitshares-core/issues/1604
+
+#### Fee Calculation
+This operation is a simple object update operation; it does not allocate new resources nor command processing logic additional to that which was already expected by the pre-existing object. A limit order on the books is created with the expectation that it will either be matched, potentially several times, or canceled, and fees to cover this storage and processing are collected during order creation. Because this operation does not fundamentally change this scenario, it charges a simple, non-refundable processing fee, and does not interact with the refundable order creation fee. The update fee is higher if the update modifies the order balance, as this increases the processing logic to affect the account balance as well as the order object.
+
+The limit order update operation may be used to "top up" an existing order and keep it on the books longer, possibly matching against more orders than would have been predicted when the order was created. This implies that for long-standing orders, the limit order update operation may constitute a discount over normal order creation, cancelation, and processing fees. The fee premium for modifying the order balance offsets this discount; however, a discount still exists for long-standing orders. Long-standing orders, however, are more valuable to the DEX than short-lived orders, as they promote market depth and high volume trading: market attributes which attract traders and more short-lived orders. The value added to the DEX as a whole by making it more economical for market makers to maintain long-standing orders significantly outweighs the discount the update operation affords to market makers; therefore, this proposal does not regard this discount on processing as detrimental to the blockchain as a whole.
```

Comment:
This is not necessarily true. The fee mechanism proposed in this BSIP encourages traders to move their partially-filled orders away from the top of the order book when the market is moving against their desired direction, and move them back when the market is moving in their desired direction, rather than cancelling it first and place a new order later. That said, it costs nodes more RAM to maintain the long-standing orders which won't fill nor help liquidity.